lib/src/particle/particle.dart

```diff
@@ -30,19 +30,19 @@ class Particle {
   Color3i color = Color3i.black;
 
   /// Use this to store application-specific body data.
-  Object userData;
+  Object? userData;
 
-  Particle(this.system, {this.group}) {
+  Particle(this.system, ParticleGroup? group) {
```

Comment:
Group shouldn't be a standard argument, since a lot of particles are created individually